validator/sawtooth_validator/consensus/proxy.py

```diff
@@ -29,6 +33,68 @@ class UnknownBlock(Exception):
     """The given block could not be found."""
 
 
+class ConsensusActivationObserver(ChainObserver):
+    """Observes chain updates and activates new engines, if they have been
```

Comment:
activating things which have been activated is confusing

Reply:
Did you mean "... if settings have been updated."?

Reply:
Updated the docstring